sites/main-site/src/content/hackathon-projects/hackathon-march-2025/coproid-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Release CoproID 2.0
```

Comment:
is this the same as nf-core/coproid? if so I would add the nf-core/ prefix?

Reply:
updated :)